tests/data_transfer_tests/services/test_inventory.py

```diff
@@ -36,3 +36,34 @@ def test_normalise_day() -> None:
     result = one == two
 
     assert result
+
+
+def test_all_devices_by_type_cache_success() -> None:
+    inventory.all_devices_by_type.cache_clear()
+    num_requests = 10
+
+    with patch("requests.get", return_value=Mock()) as _:  # act
+        for __ in range(0, num_requests):
+            inventory.all_devices_by_type(utils.DeviceType.BTF)
+
+        hits, misses, _, __ = inventory.all_devices_by_type.cache_info()
```

Comment:
I'd say this is the ACT block; perhaps do:
```python
result = inventory.all_devices_by_type.cache_info()
```
and then assert `result[0]` and `result[1]`